packages/auth/src/api/authentication/token.ts

```diff
@@ -22,15 +22,20 @@ import { querystring } from '@firebase/util';
 import {
   _getFinalTarget,
   _performFetchWithErrorHandling,
+  _performApiRequest,
+  _addTidIfNecessary,
   HttpMethod,
-  HttpHeader
+  HttpHeader,
+  Endpoint
 } from '../index';
 import { FetchProvider } from '../../core/util/fetch_provider';
 import { Auth } from '../../model/public_types';
 import { AuthInternal } from '../../model/auth';
 
-export const enum Endpoint {
-  TOKEN = '/v1/token'
+export const enum TokenType {
+  UNSPECIFIED = 'TOKEN_TYPE_UNSPECIFIED',
```

Comment:
Seems not necessary to me. We won't have to send a token with type unknown to backend.

Reply:
Done.